pkg/controller/admissionchecks/provisioning/constants.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioning
+
+const (
+	ConfigKind            = "ProvisioningRequestConfig"
+	ControllerName        = "ProvisioningRequestController"
```

Comment:
Let's namespace this: `kueue.x-k8s.io/provisioning-request`

Reply:
why?

Reply:
it's generally a good practice, in case other extensions have another form of ProvisioningRequest

Reply:
This name is part of the KEP
https://github.com/kubernetes-sigs/kueue/blob/91d5f645a326dae1ea8542879adcc62ba4db7f5d/keps/1136-provisioning-request-support/README.md?plain=1#L84-L87

Reply:
done